src/spurt/workflows/emcf/_settings.py

```diff
@@ -32,15 +33,18 @@ class SolverSettings:
         'centroid'.
     s_cost_scale: float
         Scale factor used in computing edge costs for spatial unwrapping.
+    num_parallel_tiles: int
+        Number of tiles to process in parallel. Set to 0 for all tiles.
     """
 
     t_worker_count: int = 0
-    s_worker_count: int = 1
+    s_worker_count: int = 0
```

Comment:
Do we think this is a good default to have all interferograms unwrapped at the same time? I can see how `1` would be a little annoying to make such a slow default... but is this one working for your version where you use ~4GB machines to unwrap?

Reply:
I can switch it back to 1. I would suggest that in production one must use explicit numbers for workers.